src/bpftrace.cpp

```diff
@@ -1673,6 +1686,52 @@ std::string BPFtrace::resolve_uid(uintptr_t addr) const
   return username;
 }
 
+std::string BPFtrace::resolve_timestamp(uint32_t strftime_id,
```

Comment:
Looks like this currently gives us 1s accuracy right?  I was hoping for to get closer to 100us but that should probably go  in a follow up ticket.

Reply:
Yes. If we want someting better, we may have to choose something more accurate than btime. `clock_gettime(CLOCK_REALTIME)` might help though I'm not so sure if that's what we want; and if we use it, it's likely we cannot allow `strftime` to accept user specified timestamp.